examples/README.md

```diff
@@ -135,6 +136,28 @@ before trying out the examples.
 
 - [Keep Alive](src/main/java/io/grpc/examples/keepalive)
 
+- [Cancellation](src/main/java/io/grpc/examples/cancellation)
+
+- [Custom Load Balance](src/main/java/io/grpc/examples/customloadbalance)
+
+- [Deadline](src/main/java/io/grpc/examples/deadline)
+
+- [Error Details](src/main/java/io/grpc/examples/errordetails)
+
+- [Experimental](src/main/java/io/grpc/examples/experimental)
+
+- [GRPC Proxy](src/main/java/io/grpc/examples/grpcproxy)
+
+- [Header](src/main/java/io/grpc/examples/header)
+
+- [Load Balance](src/main/java/io/grpc/examples/loadbalance)
+
+- [Multiplex](src/main/java/io/grpc/examples/multiplex)
```

Comment:
Should we add a paragraph of explanation for non-trivial examples? As done for Retrying, Health Service, etc.

Reply:
not sure on this part as this is an existing one also can see the paragraph of explanation for only 3 examples not for all which are existing previously.